packages/app/src/components/UsersHomePageFooter.tsx

```diff
@@ -52,6 +54,14 @@ export const UsersHomePageFooter = (props: UsersHomePageFooterProps): JSX.Elemen
               <span className="mx-2 ">{t('bookmark_folder.new_folder')}</span>
             </button>
           </span>
+          <span className="ml-auto pl-2 ">
+            <button
+              className={`btn btn-sm expand-compress-btn ${isExpanded ? 'active' : ''}`}
```

Comment:
please set prefix `grw-` like `grw-xxxxxxx` to make clear it's GROWI original class.